### ISSUE TITLE
Calc: In some time scroll the code app showing nextcloud files app.

### ISSUE THREAD

**Issue by matiasdelellis** [nextcloud/richdocuments#217]
### Steps to reproduce
1. Launch a calc document
2. Some random scroll

### Expected behaviour
Scrool document..

### Actual behaviour
In some time scroll the code app showing nextcloud.

![imagen](https://user-images.githubusercontent.com/733715/42456751-0de257ca-836c-11e8-8196-e995bf5fc134.png)

### Server configuration

**Operating system**: Fedora 28 but others too..

**Web server:**: Fedora 28, Apache 

**Database:**: Mariadb/mysql

**PHP version:**: 7.1

**Nextcloud version:**: Now  13.0.4, but others too..

**Updated from an older Nextcloud/ownCloud or fresh install:**: Updated.

**Where did you install Nextcloud from:** Source..

**Comment by thebearon** [comment]
What's the version in Help -> About in Collabora Online? (LOOLWSD and LOKit)

**Comment by matiasdelellis** [comment]
Just update to last:

> LOOLWSD
> 3.3.1 (git hash: 0adc773)
> 
> LOKit
> Collabora Office 5.3-53 (git hash: cf2ac11)
> 

But it happens to me before...

**Comment by matiasdelellis** [comment]
Doing more tests I notice that the movement starts when scroll on the toolbar and on the 'rows numbers'..

**Comment by thebearon** [comment]
Hm, not reproducible for me, no idea what this could be caused by.

**Comment by matiasdelellis** [comment]
Hi @thebearon 
I was finally able to establish when the problem appears.. :sweat_smile: 

 * If a file is opened from a view with few files, the file interface does not scroll.
![imagen](https://user-images.githubusercontent.com/733715/45881503-bfbc0000-bd81-11e8-9234-9022f5b716ac.png)

 * So, libreoffice online is ok.. :wink: 
![imagen](https://user-images.githubusercontent.com/733715/45881701-54266280-bd82-11e8-93ff-f7b107558849.png)

 * But if the file view has many files -i.e. Recent view-, this adds a scroll.
![imagen](https://user-images.githubusercontent.com/733715/45881578-ed08ae00-bd81-11e8-951f-c842bfe480e5.png)

 * The libreoffice interface is created inside the file view, and therefore is below the scroll window: :disappointed: 
![imagen](https://user-images.githubusercontent.com/733715/45881666-348f3a00-bd82-11e8-858e-91a4f977ab0f.png)

**Comment by st33vil** [comment]
I can confirm this issue.

**Comment by st33vil** [comment]
Is there any solution to this?
I've updated Nextcloud to v. 15, Collabora Online app to v. 3.0.6 and CODE to version 4.0 RC1. This issue stil persists.
It's a rather tedious bug, especially when hitting Ctrl-F to search inside a document - focus switches to the search box at the bottom of the spreadsheet, but you have to manually scroll browser's scrollbar to see what you are typing and to see the matching cell.

**Comment by juliushaertl** [comment]
Can you provide some details about the used browser version? I cannot reproduce this on latest firefox/chromium with Nextcloud 15 and richdocuments 3.1.0

**Comment by matiasdelellis** [comment]
Hi @juliushaertl 
>

At this point, it is independent of the browser or version. :sweat_smile: 
In my particular case, I reproduce it from firefox 52 to 64, and chrome, and chromium, Nextcloud 13 and 14 with all the updates,  and CODE installed with docker at least from 3.0 to the last 4.0RC.

I do not have NC15 installed yet. But since they now use Vue, I suppose that it change the structure a lot.. :disappointed: 

Please, try to reproduce this comment
https://github.com/nextcloud/richdocuments/issues/217#issuecomment-423517738

The fundamental thing to reproduce the bug is to launch CODE from a view with scroll..

**Comment by juliushaertl** [comment]
> I do not have NC15 installed yet. But since they now use Vue, I suppose that it change the structure a lot.. disappointed

There is no vue change in the files app with 15, so that should have no influence here.

> Please, try to reproduce this comment #217 (comment)

I did that, but unfortunately I still cannot reproduce.

Do you use the latest richdocuments version?

**Comment by st33vil** [comment]
Hi, @juliushaertl!

I have latest versions of richdocuments, Nextcloud and CODE.
I'm using Firefox (latest stable). But, for this case, i also tried Chrome 71 (fresh install, no add-ons). I even tried to run Firefox with add-ons disabled. This issue persists no matter what browser I use and is not linked with add-ons.

I tried to test a little bit more one of the files that manifest this issue and here's my conclusions:

1. File is marked as favourite.
2. When I open it within Favourites meta-folder (left side of Files app), it has described issue. My Favourites meta-folder contains more items than what can fit on the screen, so there's a browser's scrollbar.
3. When I open it in another, regular, folder (that also contains more items than what can fit on the screen and has browser's scrollbar), files opens with no issues at all.
4. When I make a search within that very same folder and afterwards open the file (I created search in a way that the test file's name matches), described issue manifests, in the same way like the one described under 2.

Hope this helps. If I can help you in any way, just ask!

Kind regards,
Kruno
